compiler/src/iree/compiler/API/CMakeLists.txt

```diff
@@ -82,6 +82,8 @@ set(_EXPORT_OBJECT_LIBS
   iree_compiler_API_Internal_IREEOptToolEntryPoint.objects
   iree_compiler_API_Internal_IREEReduceToolEntryPoint.objects
   iree_compiler_API_Internal_LLDToolEntryPoint.objects
+  # doesn't work because this target isn't known yet...
+  # iree_compiler_Codegen_Dialect_GPU_IR_IREEGPUDialect.objects
```

Comment:
This doesn't work - maybe could work - but do we want to add `IREEGPUDialect` here?